### PR TITLE
pushing up repo from project with buggy .git file

### PR DESCRIPTION
**Pull request by rlmclaughlin**

### CODE REVIEW

File: todo/src/App.js [hunk 1]
```diff
@@ -1,29 +1,50 @@
...
+import TodoForm from './Components/Form'
+
+
 
 function App() {
 
-  const [todos, setTodos] = useState(["eat","sleep","code","pray","repeat"]);  //sample todo items.  these will need to be changed in your app.  Just filler todos, although they are quite important!
+    const [todos, setTodos] = useState([]); 
+  
+    const deleteHandler = (id) => {
```
Comment (marceloarb) [inline todo/src/App.js:12]: If you create 2 equal items by mistake and try to delete one of them, they both get deleted.

File: todo/src/App.js [hunk 2]
```diff
@@ -1,29 +1,50 @@
...
+        setTodos(todos.filter(item => item.item !== id))
+    }
+  
+    const deleteAll = () => {
```
Comment (marceloarb) [inline todo/src/App.js:16]: If you have a function that is being used from only one component, that function should be there and not in the App. Like DeleteAll should be in TodoList. This breaks the single responsibility principle.

File: news/src/App.js [hunk 3]
```diff
@@ -1,29 +1,62 @@
...
+    const [headlines, setHeadlines] = useState([])
+    const [articles, setArticles] = useState([])
+    const [sidebarGenre, setSidebarGenre] = useState('')
+
+    useEffect(() => {
+        axios.get("https://api.nytimes.com/svc/topstories/v2/us.json?api-key=QRcjNGNvNxxT8GAUTmAw2ch0mByGzKBL")
+            .then(response => {
+                setHeadlines(response.data.results)
+            })
+        .catch(error => {console.log("there was an error", error)})
```
Comment (marceloarb) [inline news/src/App.js:25]: After debugging with Console.log delete.

File: news/src/App.js [hunk 4]
```diff
@@ -1,29 +1,62 @@
-import logo from './tek_logo.png';
+import React,{useEffect, useState} from 'react'
 import './App.css';
 import NavBar from './Components/NavBar';
 import Headlines from './Views/Headlines';
+import {Route} from 'react-router-dom'
 import Sports from './Views/Sports';
```
Comment (marceloarb) [inline news/src/App.js:6]: If you are not using delete.

**Review by marceloarb — APPROVED** [review]
Looks pretty good. Good job

**Comment by rlmclaughlin** [comment]
Hey Marcelo. My internet is down and my phone isn’t doing good either. Trying to create a hotspot but It hasn’t been fast enough to even open a page. Just giving you a heads up. I’ll keep trying but I don’t know when or if I’ll make it.

**Pull request merged**